README.md

```diff
@@ -29,6 +29,7 @@ cd $CMSSW_BASE/src
 git clone git@github.com:susy2015/ZInvisible.git
 cd ZInvisible/Tools
 ```
+mkdir plots
```

Comment:
can you put this within the code section (before ```)

Reply:
that way it will be formatted the same way; we use this format for terminal commands

Reply:
I move it already

Reply:
I see it now. Thanks!